File: Submissions/T16/H3.c
#include <stdio.h>

int faktorial(int n) {
    if (n <= 1) return 1;
    return n * faktorial(n - 1);
}

int pangkat(int a, int b) {
    if (b == 0) return 1;
    return a * pangkat(a, b - 1);
}

int main() {
    int n, a, b;
    
    printf("Masukkan n (faktorial): ");
    scanf("%d", &n);
    printf("%d! = %d\n", n, faktorial(n));
    
    printf("Masukkan a dan b: ");
    scanf("%d %d", &a, &b);
    printf("%d^%d = %d\n", a, b, pangkat(a, b));
    
    return 0;
}

File: Submissions/T07/H3.py

def faktorial(n):
    if n == 0 or n == 1:
        return 1
    return n * faktorial(n - 1)


def pangkat(a, b):
    if b == 0:
        return 1
    return a * pangkat(a, b - 1)


n = int(input("Masukkan n (faktorial) : "))
a, b = map(int, input("Masukkan a dan b : ").split())

print(f"{n}! = {faktorial(n)}")
print(f"{a}^{b} = {pangkat(a, b)}")